lib/DxilDia/DxilDiaSymbolManager.cpp

```diff
@@ -1140,13 +1143,23 @@ void dxil_dia::hlsl_symbols::SymbolManagerInit::TypeInfo::Embed(const TypeInfo &
   for (const auto &E : TI.GetLayout()) {
     m_Layout.emplace_back(E);
   }
+  uint64_t alignmentInBytes = TI.GetAlignmentInBytes();
+  if (alignmentInBytes != 0 && 
+      m_dwCurrentSizeInBytes % alignmentInBytes != 0) {
+    m_dwCurrentSizeInBytes += alignmentInBytes - (m_dwCurrentSizeInBytes % alignmentInBytes);
```

Comment:
Isn't this basically llvm::RoundUpToAlignment? Same with in TypeInfo::AddBasicType.

Reply:
So it is! Didn't know about that. Changed to use it.